### PR TITLE
[PXCT-324] Nicla Voice: Glass-Breaking Detector Application Note

### PR DESCRIPTION
**Pull request by ArduinoBot**
## What This PR Changes
Adds glass breaking detector with Edge Impulse and Nicla Voice application note

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md [hunk 1]
```diff
@@ -0,0 +1,838 @@
...
+
+After the build is complete, the firmware files will automatically download. You can retrieve them later from the **Latest build** tab in the **Deployment** section if needed.
+
+![Nicla Voice Firmware Build Files](assets/edge-fw-build.png)
+
+The downloaded firmware comes as a compressed file. Extract it to your preferred directory. You will find several files in the extracted folder, including a directory named **`ndp120`**.
+
+This folder contains the NDP120 processor firmware and the trained machine learning model. There are four scripts for each supported operating system:
+
+- **flash_<os>:** Runs the MCU firmware and model flash scripts.
```
Comment (mcmchris) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md:202]: Using <> in the markdown is problematic, this is why the local preview doesn't work also the github preview, I changed to this: and it fixes all the problems. 

- **flash_(os):** Runs the MCU firmware and model flash scripts.
- **flash_(os)_mcu:** Flashes only the MCU firmware.
- **flash_(os)_model:** Flashes the NDP firmware, DSP firmware (if not present), and updates the model.
- **format_(os)_ext_flash:** Erases the external flash memory.

**`(os)`** refers to the operating system: linux, mac, or windows.

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md [hunk 2]
```diff
@@ -0,0 +1,838 @@
...
+title: 'Glass-Breaking Detection Using Edge Impulse & Nicla Voice'
+description: "This application note describes creating a glass-breaking detection system using the Arduino Nicla Voice, Machine Learning Tools, and the Edge Impulse platform."
+difficulty: intermediate
+tags:
+  - Nicla
+  - Edge Impulse
+  - Machine Learning Tools
+  - Audio Classification
+  - Sound Detection
+author: 'Taddy Ho Chung & Christopher Mendez'
```
Comment (mcmchris) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md:11]:
```suggestion
author: 'Taddy Ho Chung & Christopher Méndez'
```

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md (hunk 1 above)
Comment (TaddyHC) [reply content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md:205]: ah okay, thanks! Applying the changes

**Comment by TaddyHC** [comment]
@mcmchris thanks for the feedback and as well for the assets! Feedback applied and updated

**Review by mcmchris — APPROVED** [review]

**Comment by mcmchris** [comment]
LGTM

**Pull request merged**